backend/app/services/admin/user_management_service.py

```
@@ -0,0 +1,117 @@
+    async def reset_verification(self, user_id: str) -> bool:
+        try:
+            supabase = get_supabase_client()
+            await supabase.table("users").update({
+                "github_id": None,
+                "github_username": None,
+                "is_verified": False,
+                "verification_token": None,
+                "updated_at": datetime.now().isoformat()
+            }).eq("discord_id", user_id).execute()
+            logger.info(f"Reset verification for user {user_id}")
+            return True
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)